x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -423,7 +423,18 @@ export class FleetPlugin
           summary: 'Fleet is setting up',
         });
 
-        await plugins.licensing.license$.pipe(take(1)).toPromise();
+        // We need to wait for the licence feature to be available,
+        // to have our internal saved object client with encrypted saved object working properly
+        await plugins.licensing.license$
+          .pipe(
+            filter(
+              (licence) =>
+                licence.getFeature('security').isEnabled &&
+                licence.getFeature('security').isAvailable
```

Comment:
I am wondering if we need any special error handling here, could it happen that security feature is not enabled at all?

Reply:
I do not think it's possible to get security disabled with Fleet